jvb/src/main/kotlin/org/jitsi/videobridge/cc/vp9/Vp9QualityFilter.kt

```diff
@@ -110,7 +111,9 @@ internal class Vp9QualityFilter(parentLogger: Logger) {
                I think this shouldn't be a problem? */
             getSidFromIndex(incomingIndex) == getSidFromIndex(externalTargetIndex)
         }
-        return AcceptResult(accept, mark)
+        val isResumption = (prevIndex == SUSPENDED_INDEX && currentIndex != SUSPENDED_INDEX)
+        if (isResumption) assert(accept) // Pretty sure this has to be true?
```

Comment:
nit: remove the comment
I don't see how you can have a resumption and then don't accept the frame either.